gapic/src/main/com/google/gapid/perfetto/views/SlicesSelectionView.java

```diff
@@ -104,37 +104,47 @@ private void setSingleSliceView(State state, SliceTrack.Slices slice) {
     if (slice instanceof GpuSlices) {
       renderStageInfo = ((GpuSlices)slice).getRenderStageInfoAt(0);
     }
+    ImmutableMap.Builder<String, String> propsBuilder = ImmutableMap.builder();
     if (renderStageInfo != null && renderStageInfo != RenderStageInfo.EMPTY) {
-      ImmutableMap.Builder<String, String> propsBuilder = ImmutableMap.builder();
+      ImmutableMap.Builder<String, String> vkPropsBuilder = ImmutableMap.builder();
       if (renderStageInfo.frameBufferHandle != 0) {
         if (renderStageInfo.frameBufferName.isEmpty()) {
-          propsBuilder.put("VkFrameBuffer:", String.format("0x%08X", renderStageInfo.frameBufferHandle));
+          propsBuilder.put("Render Target:", String.format("0x%08X", renderStageInfo.frameBufferHandle));
         } else {
-          propsBuilder.put("VkFrameBuffer:", renderStageInfo.frameBufferName + " <" + String.format("0x%08X", renderStageInfo.frameBufferHandle) + ">");
+          propsBuilder.put("Render Target:", renderStageInfo.frameBufferName + " <" + String.format("0x%08X", renderStageInfo.frameBufferHandle) + ">");
         }
       }
       if (renderStageInfo.renderPassHandle != 0) {
         if (renderStageInfo.renderPassName.isEmpty()) {
-          propsBuilder.put("VkRenderPass:", String.format("0x%08X", renderStageInfo.renderPassHandle));
+          vkPropsBuilder.put("VkRenderPass:", String.format("0x%08X", renderStageInfo.renderPassHandle));
         } else {
-          propsBuilder.put("VkRenderPass:", renderStageInfo.renderPassName + " <" + String.format("0x%08X", renderStageInfo.renderPassHandle) + ">");
+          vkPropsBuilder.put("VkRenderPass:", renderStageInfo.renderPassName + " <" + String.format("0x%08X", renderStageInfo.renderPassHandle) + ">");
         }
       }
       if (renderStageInfo.commandBufferHandle != 0) {
         if (renderStageInfo.commandBufferName.isEmpty()) {
-          propsBuilder.put("VkCommandBuffer:", String.format("0x%08X", renderStageInfo.commandBufferHandle));
+          vkPropsBuilder.put("VkCommandBuffer:", String.format("0x%08X", renderStageInfo.commandBufferHandle));
         } else {
-          propsBuilder.put("VkCommandBuffer:", renderStageInfo.commandBufferName + " <" + String.format("0x%08X", renderStageInfo.commandBufferHandle) + ">");
+          vkPropsBuilder.put("VkCommandBuffer:", renderStageInfo.commandBufferName + " <" + String.format("0x%08X", renderStageInfo.commandBufferHandle) + ">");
         }
       }
 
       if (renderStageInfo.submissionId != 0) {
         propsBuilder.put("SubmissionId:", Long.toString(renderStageInfo.submissionId));
       }
 
+      ImmutableMap<String, String> vkProps = vkPropsBuilder.build();
+      if (!vkProps.isEmpty()) {
+        withLayoutData(createBoldLabel(main, "Vulkan Info:"),
+            withSpans(new GridData(), 2, 1));
+        vkProps.forEach((key, value) -> {
+          createSelectableLabel(main, key);
+          createSelectableLabel(main, value);
+        });
+      }
```

Comment:
Nit: Just personal opinion, would we like to render general info first and vk info the next? It looks more straight forward to me to see general things first.